gradle.properties

```diff
@@ -24,10 +24,11 @@ jsonUnitVersion=2.38.0
 kotlinCoroutinesVersion=1.8.1
 kotlinLoggingVersion=2.1.23
 mockkVersion=1.13.5
-springMockkVersion=3.1.2
+springMockkVersion=4.0.2
```

Comment:
[Version 4.x of SpringMockK: compatible with Spring Boot 3.x, Java 17+](https://github.com/Ninja-Squad/springmockk?tab=readme-ov-file#versions-compatibility)